pkg/controller/manage/manage_controller.go

```diff
@@ -138,6 +141,9 @@ func (c *KmeshManageController) handlePodAdd(obj interface{}) {
 
 	namespace, err := c.namespaceLister.Get(newPod.Namespace)
 	if err != nil {
+		if apierrors.IsNotFound(err) {
+			return
```

Comment:
Is there any change from before? Other than one less line of logging.

Reply:
?